docs/SPEC.md

```diff
@@ -2161,7 +2157,8 @@ from(bucket:"telegraf/autogen")
                 r.service == "app-server")
     |> range(start:-12h)
     // Square the value
-    |> map(fn: (r) => r._value * r._value)
+    // The output table has all column from the input table because of the use of `with`.
+    |> map(fn: (r) => {r with _value: r._value * r._value})
```

Comment:
I think we still require parenthesis around the object literal syntax when using a function.